src/main/resources/db/changelog/db-changelog-master.xml

```diff
@@ -8,18 +8,20 @@
    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
 
     <changeSet id="users-table-init" author="Ehsan">
-        <createTable tableName="users">
+        <createTable tableName="table_user">
```

Comment:
I don't like the name 'table_xxx'. Do not use this pattern for naming the tables.
You can read more about the good patterns for db table namings on [https://www.sqlshack.com/learn-sql-naming-conventions/](url)